src/commands.ts

```diff
@@ -64,37 +64,50 @@ export function registerToolchainCommands(
     ];
 }
 
+export enum COMMANDS {
```

Comment:
praise: great idea 👍

Reply:
Agreed this is a good idea! A nit though, stylistically having this in all caps is inconsistent, I believe it should just be named `Commands`